src/domain/settlement/index.js

```diff
@@ -99,12 +93,12 @@ module.exports = {
       for (const key of Object.keys(settlementWindowsList)) {
         const windowContentRecord = await SettlementWindowContentModel.getBySettlementAndWindowId(settlementId, settlementWindowsList[key].id)
```

Comment:
yes it returns a list, but since the window is defined by the `key` const, it always returns one result. Event with multiple settlement windows in single settlement.